Code/Tools/AssetProcessor/native/unittests/AssetProcessingStateDataUnitTests.cpp

```diff
@@ -10,86 +10,12 @@
 #include <native/unittests/AssetProcessorUnitTests.h>
 #include <native/unittests/UnitTestRunner.h> // for the assert absorber.
 
+using namespace AzToolsFramework::AssetDatabase;
+
 namespace AssetProcessor
 {
-    class AssetProcessingStateDataUnitTest
-        : public UnitTest::AssetProcessorUnitTestBase
-    {
-    protected:
-        void SetUp() override
-        {
-            UnitTest::AssetProcessorUnitTestBase::SetUp();
-
-            EXPECT_TRUE(m_connection.DataExists());
-            EXPECT_TRUE(m_connection.OpenDatabase());
-
-        }
-
-        void TearDown() override
-        {
-            m_connection.CloseDatabase();
-
-            UnitTest::AssetProcessorUnitTestBase::TearDown();
-        }
-
-        AssetProcessor::AssetDatabaseConnection m_connection;
-    };
-
-    TEST_F(AssetProcessingStateDataUnitTest, DataTest_ValidDatabaseConnectionProvided_OperationsSucceed)
+    namespace
```

Comment:
strictly speaking, O3DE coding practices do not use anonymous namespaces
could this be "namespace Internal" or "namespace Testing" ?

Reply:
Yeah, I can make the change.